projects/HieraSeg/decode_head/sep_aspp_contrast_head.py

```diff
@@ -146,10 +154,45 @@ def predict_by_feat(self, seg_logits: Tensor,
 
         return seg_logit
 
-    def losses(self, results, seg_label):
-        """Compute segmentation loss."""
-        seg_logit_before = results[0]
-        embedding = results[1]
+    def loss(self, inputs: Tuple[Tensor], batch_data_samples: SampleList,
+             train_cfg: ConfigType) -> dict:
+        """Forward function for training.
+
+        Args:
+            inputs (Tuple[Tensor]): List of multi-level img features.
+            batch_data_samples (list[:obj:`SegDataSample`]): The seg
+                data samples. It usually includes information such
+                as `img_metas` or `gt_semantic_seg`.
+            train_cfg (dict): The training config.
+
+        Returns:
+            dict[str, Tensor]: a dictionary of loss components
+        """
+        seg_logits = self.forward(inputs)
+        losses = self.loss_by_feat(seg_logits, batch_data_samples)
+        return losses
```

Comment:
I suggest removing these codes since it's the same as the base class.

Reply:
I agree with you😀